src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -24,18 +24,56 @@
 namespace Tizen.NUI.Components
 {
     /// <summary>
-    /// The Navigator is a class which navigates pages with stack methods such
-    /// as Push and Pop.
+    /// TransitionFinishedEventArgs is a class to record page TransitionFinished event arguments which will be sent to user.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class TransitionFinishedEventArgs : EventArgs
```

Comment:
Is there a plan to add properties in the class? If not, I don't think this class is needed. You may just use `EventArgs` directly.

Reply:
Thank you.